protocol-definitions/custom/Custom.yaml

```diff
@@ -1499,3 +1499,15 @@ customTypes:
         type: String
         since: 2.7
         nullable: true
+  - name: Version
+    since: 2.7
+    returnWithFactory: true
+    params:
```

Comment:
Very similar to `MemberVersion` (assume patch field is 0). Do we really need yet another new custom type?

Reply:
The cluster version is not `MemberVersion`, and I am not introducing another type, just adding an existing type to the client protocol. The WAN version is also a `Version`, i.e. it consists of major and minor versions only, so `Version` may have other usages in the _near_ future, considering that we try to send more cluster information to the clients.